tox.ini

```diff
@@ -10,6 +11,10 @@ commands =
     coverage report --fail-under 100
     pre-commit install
 
+[testenv:venv]
+envdir = venv-{[tox]project}
+commands =
```

Comment:
let's put this below the actually used environments (also conveniently alphabetical!)